FluidNC/src/Protocol.cpp

```diff
@@ -199,7 +199,7 @@ void protocol_main_loop() {
                 break;
             }
 #ifdef DEBUG_REPORT_ECHO_RAW_LINE_RECEIVED
-            report_echo_line_received(line, *chan);
+            report_echo_line_received(line, *chan); 
```

Comment:
The trailing space at the end of this line suggests that the code has not been processed by clang-format .  We require all submissions to be so formatted.